mappings/bloblang/AWS/CloudTrail/cloudtrail_authentication.blobl

```diff
@@ -4,61 +4,96 @@ meta ocsf_topic = "authentication"
 root.class_uid = 3002
 root.class_name = "Authentication"
 root.category_uid = 3
-root.category = "Identity & Access Management Category"
+root.category_name = "Identity & Access Management"
 root.cloud.provider = "AWS"
 root.severity_id = 1
 root.severity = "Informational"
-root.status_id = match this.ConsoleLogin.responseElements.ConsoleLogin {
+root.status_id = match this.responseElements.ConsoleLogin|serviceEventDetails.UserAuthentication {
```

Comment:
<img width="226" alt="image" src="https://github.com/ocsf/examples/assets/13574947/67da443c-bb23-4955-acb1-a60fb9308113">
We do get this field sometimes @floydtree is this normal ?

Reply:
Interesting, do you have a full event that you can share? I haven't seen such a struct, curious to see which API call it was.